.ort.yml

```diff
@@ -1,36 +1,28 @@
 curations:
   license_findings:
-    - path: "xyz-models/src/main/java/com/here/xyz/models/hub/Space.java"
-      reason: "DATA_OF"
-      comment: "The file contains all possible license identifiers for the content of a space."
-      concluded_license: "Apache-2.0"
-    - path: "xyz-hub-service/src/main/resources/openapi.*"
-      reason: "DATA_OF"
-      comment: "The files found in this directory are part of the OpenAPI specification which contains various license keywords, see e.g. \
-        https://github.com/heremaps/xyz-hub/blob/master/xyz-hub-service/src/main/resources/webroot/openapi.yaml#L1172-L1205."
-      concluded_license: "Apache-2.0"
+    - path: "here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/License.java"
+      start_lines: "56"
+      line_count: 1
+      detected_license: "OSL-3.0"
+      reason: "CODE"
+      comment: "The scanner matches an enumerator `OSL-3.0`."
+      concluded_license: "NONE"
+    - path: "here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/License.java"
+      start_lines: "62"
```

Comment:
Not sure exactly how these license scanners work, but telling them which exact line to find an array entry would be susceptible to changes simply from editing the file. Would be great if there is a way to avoid this.

Reply:
Licensing exclusion is usually safer to be on a conservative side. It is better to have minimum possible exclusion than keeping it wide open. Also, this file is not expected to change that often. (not changed even once during our entire development cycle ;-) )